site/package.json

```diff
@@ -33,7 +33,7 @@
     "react-dom": "18.2.0",
     "react-tooltip": "4.2.21",
     "swr": "2.2.5",
-    "tailwindcss": "2.2.19",
+    "tailwindcss": "3.4.18",
```

Comment:
Due to this upgrade, we need to replace `overflow-ellipsis` with `text-ellipsis`

It seems the only usage is in `site/pages/[locale]/token-revokes.js` line 293. See [class name changes](https://v3.tailwindcss.com/docs/upgrade-guide#class-name-changes)